torch/_inductor/codegen/wrapper.py

```diff
@@ -666,9 +671,23 @@ def generate_user_defined_triton_kernel(
         stream_name = self.write_get_raw_stream(
             V.graph.scheduler.current_device.index, V.graph
         )
-        self.writeline(
-            f"{kernel_name}.run({', '.join(args)}, grid={grid}, stream={stream_name})"
-        )
+
+        if profiler._capture_generated_kernel:
```

Comment:
@eellison here we're depending on some global profiler state. Is that fine? (and is the codegen change below fine?)

semantics: depending on whether this global flag is set in profiler, we may choose to add these profiling calls into the generated triton code.

(maybe should this config go in `torch._inductor.config`?)

Reply:
Yea if this is an inductor only flag let's just put it in the `torch._inductor.config` (which will be accounted for in things like inductor graph caching).